internal/ips/retrieve.go

```diff
@@ -101,11 +100,12 @@ func (c *Client) Retrieve() *cobra.Command {
 				return c.Out.Output(ip, header, &data)
 			}
 
-			ips, err := pager.GetAllIPReservations(c.IPService, projectID, inc, exc, types)
+			// Don't have a paginator for this one
+			resp, _, err := c.IPService.FindIPReservations(context.Background(), projectID).Include(inc).Exclude(exc).Types(types).Execute()
```

Comment:
We're not using the Servicer.Excludes/Includes functions in some of these pagination locations. This is needed to ensure that the `--include` and `--exclude` arguments from the CLI make it into the request.

This PR has me wondering if we should introduce `--page`, `--per-page` as globals, like `include` and `exclude`. These would also be exposed through the Servicer to pass the desired pagination settings into the request. This would allow a user to skip the full list or paginate on their own terms. Sort options (`sort_by`, `sort_dir`) would also make sense with `page`.

Reply:
I wired up the missing includes/excludes flags for the commands that are touched in this PR, but there may be others that were missed in the SDK conversion.

IMO once someone wants deep control of pagination they're probably better off using the SDK directly, rather than coding around the CLI, but we could open an issue for that feature and solicit interest/feedback from users.